src/hrflow_connectors/connectors/hrflow/warehouse/tracking.py

```diff
@@ -0,0 +1,313 @@
+import typing as t
+from enum import Enum
+from logging import LoggerAdapter
+
+from hrflow import Hrflow
+from pydantic import Field
+
+from hrflow_connectors.connectors.hrflow.schemas import HrflowTracking
+from hrflow_connectors.core import (
+    DataType,
+    FieldType,
+    ParametersModel,
+    ReadMode,
+    Warehouse,
+    WarehouseReadAction,
+    WarehouseWriteAction,
+)
+
+
+class Role(str, Enum):
+    recruiter = "recruiter"
+    candidate = "candidate"
+    employee = "employee"
+    manager = "manager"
+
+
+class ReadTrackingParameters(ParametersModel):
+    api_secret: str = Field(
+        ...,
+        description="X-API-KEY used to access HrFlow.ai API",
+        repr=False,
+        field_type=FieldType.Auth,
+    )
+    api_user: str = Field(
+        ...,
+        description="X-USER-EMAIL used to access HrFlow.ai API",
+        field_type=FieldType.Auth,
+    )
+
+    role: Role = Field(
+        ...,
+        description="Filter by role(Role of the user rating the job)",
+        field_type=FieldType.QueryParam,
+    )
+    actions: t.Optional[t.List[str]] = Field(
+        None,
+        description=(
+            "Filter by actions\nThe 'action' refers to a unique identifier for a"
+            " profile or job stage. This can be a specific stage ID within a CRM, ATS,"
+            ' or Job site.Examples of such stages include "view," "apply," "hire," or'
+            " any other stage relevant to your system."
+        ),
+        field_type=FieldType.QueryParam,
+    )
+    source_keys: t.Optional[t.List[str]] = Field(
+        None,
+        description=(
+            "Filter by source keys\nTo filter by a specific list of sources, include"
+            " the parameter source_keys and leave source_key empty,"
+            " profile_key//profile_reference) empty."
```

Comment:
`" and leave 'source_key', 'profile_key' and 'profile_reference' empty`  
Tu peux mettre un `and` à la place de `//` vu que a priori si j'ai bien compris au final il faut effectivement que les deux soit empty même si en pratique c'est que l'un ou l'autre qui est susceptible de pas être empty. 

Mais le truc c'est que le symbole `//` je suis pas au courant que ca veut dire _ou_ . Tu aurais mis `||` ou `|` au moins le premier c'est ou en Javascript | Typescript et le deuxième c'est ou dans RegExp

Reply:
Pareil a chaque fois ou tu as utilisé `//` pour dire _ou_ dans ce fichier. Utiliser plutot `or` , `||` ou `|` 

Sauf si tu sais que `//` est communément compris comme voulant dire _ou_